### PR TITLE
feat: install dotnet local package if github sha is present

### PR DESCRIPTION
**Pull request by jsalaber**
# Summary

Added `SDK_GITHUB_SHA` to docker image, install local package when it's present.

### CODE REVIEW

File: proxies/dotnet/install.sh [hunk 1]
```diff
@@ -1,13 +1,36 @@
+if [ -n "$SDK_GITHUB_SHA" ]; then
```
Comment (ajwootto) [inline proxies/dotnet/install.sh:1]: werent you also checking the SDKS_TO_TEST variable on the other platforms?

File: proxies/dotnet/install.sh [hunk 2]
```diff
@@ -1,13 +1,36 @@
...
+    # changing directory so dotnet doesn't run again when starting up the server
+    cd ..
+    git clone https://github.com/DevCycleHQ/dotnet-server-sdk.git
```
Comment (ajwootto) [inline proxies/dotnet/install.sh:4]: shouldnt this be using the github sha?

Comment (jsalaber) [reply proxies/dotnet/install.sh:4]: oh yes my mistake

Comment (ajwootto) [reply proxies/dotnet/install.sh:4]: bump

Comment (ajwootto) [reply proxies/dotnet/install.sh:4]: oh nvm

**Review by ajwootto — APPROVED** [review]

**Pull request merged**